Index: scantailor-EXPERIMENTAL_2016_01_13/filters/page_split/OptionsWidget.cpp
===================================================================
--- scantailor-EXPERIMENTAL_2016_01_13.orig/filters/page_split/OptionsWidget.cpp
+++ scantailor-EXPERIMENTAL_2016_01_13/filters/page_split/OptionsWidget.cpp
@@ -28,6 +28,7 @@
 #include <QPixmap>
 #include <boost/foreach.hpp>
 #include <assert.h>
+#include <QButtonGroup>
 
 namespace page_split
 {
